vm/vmIfAndSwitch_test.go

```diff
@@ -54,6 +54,29 @@ func TestSwitch(t *testing.T) {
 	os.Setenv("ANKO_DEBUG", "1")
 	tests := []testStruct{
 		// TODO: add tests
+
+		{script: `switch {}`, parseError: fmt.Errorf("syntax error")},
+		{script: `a=1; switch a; {}`, parseError: fmt.Errorf("syntax error")},
+
+		{script: `a=1; switch a {}`, runOutput: int64(1)},
+
+		{script: `a=1; switch a {case 1: return a}`, runOutput: int64(1)},
```

Comment:
please add test for `switch a { case: return a}`

Reply:
ok, and add 
```go
{script: `a=1; switch a {case: return a}`, parseError: fmt.Errorf("syntax error")},

{script: `a=99; switch a {default: return 'default'}`, runOutput: "default"},
```